contracts/standard/arbitration/ArbitrableTransaction.sol

```diff
@@ -24,26 +24,44 @@ import "./TwoPartyArbitrable.sol";
      *  @param _hashContract Keccak hash of the plain English contract.
      *  @param _timeout Time after which a party automatically loose a dispute.
      *  @param _partyB The recipient of the transaction.
+     *  @param _amountOfChoices The number of ruling options available.
```

Comment:
Why would ArbitrableTransaction have this parameter? It's always between two parties.

Reply:
You are right. Fixed.